openbas-api/src/main/java/io/openbas/rest/payload/service/PayloadService.java

```diff
@@ -126,6 +138,12 @@ private Contract buildContract(
     ContractExpectations expectationsField = expectations();
     ContractDef builder = contractBuilder();
     builder.mandatoryGroup(assetField, assetGroupField);
+
+    if (injector.getType().equals("openbas_implant") && payload.getType().equals("Command")) {
```

Comment:
**Question**
I am not sure this condition is necessary.
If you are here, so it's a custom payload right ?
So, it will be run thanks to openbas_implant